src/peft/peft_model.py

```diff
@@ -199,6 +203,25 @@ def save_pretrained(
             os.makedirs(output_dir, exist_ok=True)
 
             if safe_serialization:
+                # Safetensors does not allow tensor aliasing.
```

Comment:
This block is basically copied from https://github.com/huggingface/transformers/blob/main/src/transformers/modeling_utils.py#L2111-L2134, right? Let's add a comment.

Reply:
Ok perfect!